src/framework/shortcuts/view/mididevicemappingmodel.cpp

```diff
@@ -51,11 +55,12 @@ inline std::vector<std::string> allMidiActions()
         "pad-note-16",
         "pad-note-32",
         "pad-note-64",
+        "undo",
```

Comment:
Isn't it a bit of an illogical order to have undo between the note input related actions?

Reply:
good question, @jessjwilliamson FYI

Reply:
@Eism This was just reflection from MS3. Also, I realised there are some actions missing from the list - Stop and Real time advance

Reply:
Real-time advance has not yet been ported from MU3

Reply:
I will add stop action
@jessjwilliamson what about the ordering?

Reply:
Hi, really sorry, but the 'Stop' action is no use as I remember we replaced this with having a play and pause button instead. The stop button is not even in the play panel.